test/controllers/EventReportControllerSpec.scala

```diff
@@ -173,23 +176,53 @@ class EventReportControllerSpec extends AsyncWordSpec with Matchers with Mockito
 
     "throw validation exception when validation errors response" in {
       val controller = application.injector.instanceOf[EventReportController]
-      val listErrors: List[ErrorReport] = List(
-        ErrorReport("instance1", "errors1"),
-        ErrorReport("instance2", "errors2")
-      )
 
-      when(mockJSONPayloadSchemaValidator.validateJsonPayload(any(), any())) thenReturn Left(listErrors)
+      when(mockJSONPayloadSchemaValidator.validatePayload(any(), any(), any()))
+        .thenReturn(Failure(new Exception("Test")))
 
-      recoverToExceptionIf[EventReportValidationFailureException] {
+      recoverToExceptionIf[Exception] {
```

Comment:
would it be better to test this with expected exception class ? (EventReportValidationFailureException)
Because Exception could be thrown for any other reason